composer.json

```diff
@@ -82,7 +82,14 @@
             "symfony/thanks": false,
             "php-http/discovery": true
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "audit": {
+            "ignore": [
+                "PKSA-gs8r-6kz6-pp56",
+                "PKSA-gnn4-pxdg-q76m"
```

Comment:
Where did you get these keys? Because I assume you mean these CVEs: `CVE-2025-31481`, `CVE-2025-31485`, but I can't find them with these keys 🤔